### ISSUE TITLE
Export files available for  quantitative /qualitative  expression

### ISSUE THREAD

**Issue by pombase-admin** [pombase/pombase-chado#341]
in the build
http://curation.pombase.org/dumps/latest_build/
can we also make the modification and the qualitative and quantitative  expression  data available as a single file ?

Once this is done we should think about adding these as a PomBase file download option

v

Original comment by: ValWood

**Comment by kimrutherford** [comment]
The modification exporting code is done and a file of modifications is now exported after each load.

**Comment by ValWood** [comment]
we don't need to export the expression date files until anyone asks for it...

**Comment by ValWood** [comment]
since no one asked yet

**Comment by ValWood** [comment]
duplicate of https://github.com/pombase/pombase-chado/issues/895
